### DISCUSSION TITLE
Additional buttons in windows

### DISCUSSION THREAD

**Discussion by sheldon-walters** [nicolaasburgers/definition-library-releases#16] [Ideas]
Consider the addition of a 'Publish All' function, in the publish window.
Consider the addition of a repo/path filter in the Library search window

**Comment by nicolaasburgers** [comment]
There is a new repo filter in the latest version 0.1.4:

![image](https://github.com/nicolaasburgers/definition-library-releases/assets/3597587/e049165f-6668-460f-933f-5d7f82342814)
